src/System.IO.Pipes/src/System/Net/Sockets/UnixDomainSocketEndPoint.cs

```diff
@@ -9,16 +10,27 @@ namespace System.Net.Sockets
     /// <summary>Represents a Unix Domain Socket endpoint as a path.</summary>
     internal sealed class UnixDomainSocketEndPoint : EndPoint
     {
-        private const int MaxPathLength = 92;   // sockaddr_un.sun_path at http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_un.h.html
-        private const int PathOffset = 2;       // = offsetof(struct sockaddr_un, sun_path). It's the same on Linux and OSX
-        private const int MaxSocketAddressSize = PathOffset + MaxPathLength;
-        private const int MinSocketAddressSize = PathOffset + 2; // +1 for one character and +1 for \0 ending
         private const AddressFamily EndPointAddressFamily = AddressFamily.Unix;
 
         private static readonly Encoding s_pathEncoding = Encoding.UTF8;
+        private static readonly int s_pathOffset;
+        private static readonly int s_maxPathLength;
+        private static readonly int s_addressSize;
```

Comment:
Maybe rename to s_nativePathOffset and s_nativeAddressSize then.  It's just weird to me, later in the code, to see if (socketAddress.Size > s_addressSize), without some qualification to show that these are in fact different concepts (the actual address size and the maximum/native underlying storage).  It's s nit, to be sure. :)